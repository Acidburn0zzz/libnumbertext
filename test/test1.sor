# line separators

1 one; 2 two
3 three

# comments
4 four # ;5 foo

# boundary

^5$ five
^5 fif
1(5) $1teen # fifteen

^0$ zero
1(0) ten$1 # ten

8$ eight
8 eigh
1(8) $1teen # eighteen

# back-references

6 six
7 seven
9 nine

1(4) $1teen
1(6) $01teen
1(7) $(\1)teen
1(9) $(\01)teen

# conditional text

2(\d) twenty[-$1] # twenty, twenty-one
3([08]) thirty[-$(\1)] # thirty, thirty-eight
4([05]) [$1und]fierzig # fierzig, fifundfierzig
