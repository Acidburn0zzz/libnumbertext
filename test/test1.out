one
two
three
four
five
fifteen
zero
ten
eight
eighteen
six
seven
nine
fourteen
sixteen
seventeen
nineteen
twenty
twenty-one
thirty
thirty-eight
fierzig
fifundfierzig
